client/src/pages/Main.js

```diff
@@ -0,0 +1,76 @@
+import React, {Component, useState} from "react";
+import Home from "./Home";
+import Student from "./Student";
+import Course from "./Course";
+import ViewSchedule from "./ViewSchedule";
+import Confirmation from "./Confirmation";
+
+
+class Main extends Component  {
+    constructor(args){
+        super(args);
+        this.state = {
+          values: {
+                studentName: '',
+                classTimes:'',
+                hoursAvail:0,
+                coursePref:'',
+                facultyPref:'',
+                officeHours: 0
+          },
+          step: 0
+        }
+    }
+    
+      saveValues(values) {
+        this.setState({
+          values: Object.assign({}, this.state.values, values)
+        })
+      }
+    
+      nextStep() {
+        this.setState({
+          step: this.state.step +1
+        })
+      }
+    
+      previousStep() {
+        this.setState({
+          step: this.state.step -1
+        })
+      }
+
```

Comment:
Switch case allows user to switch between home page, student page, course page, and confirmation.